.travis.yml

```diff
@@ -75,7 +75,7 @@ script:
   - sh -c "! grep '^\[' /tmp/cppcheck.cppcheck"
 # check naming conventions
   - ${CPPCHECK} -j 2 --dump -q gui lib
-  - python addons/naming.py --private-member-variable='m[A-Z].*' gui/*.cpp.dump lib/*.cpp.dump
```

Comment:
Thanks!

Isn't it easier to just
````shell
find lib gui -name "*.dump*" | xargs [...]
````
or am I missing something?

Reply:
With `find lib gui -maxdepth 1 -name "*.dump"` i get the same files, so that should work.